crates/relayer/src/config.rs

```diff
@@ -50,6 +53,54 @@ impl Display for GasPrice {
     }
 }
 
+impl TryFrom<String> for GasPrice {
```

Comment:
It would more idiomatic to implement `std::str::FromStr` instead of `TryFrom<String>`